Storage account has tags (#648)
aquasecurity/cloud-security-remediation-guides: en/azure/storageaccounts/storage-account-has-tags.md

[![CloudSploit](https://cloudsploit.com/img/logo-new-big-text-100.png "CloudSploit")](https://cloudsploit.com)

# AZURE / Storage Accounts / Storage Account Has Tags

## Quick Info

| | |
|-|-|
| **Plugin Title** | Storage Account Has Tags |
| **Cloud** | AZURE |
| **Category** | Storage Accounts |

> | **Description** | Ensures that Azure Storage Account Has Tags associated |

Reviewer: ```suggestion
| **Description** | Ensure that Azure Storage accounts have tags associated. |
```

| **More Info** | Tags help you to group resources together that are related to or associated with each other. It is a best practice to tag cloud resources to better organize and gain visibility into their usage. |
| **AZURE Link** | https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/tag-resources |

> | **Recommended Action** | 'Modify storage account and add tags |

Reviewer: ```suggestion
| **Recommended Action** | Modify storage account and add tags. |
```

## Detailed Remediation Steps

> 

Reviewer: ```suggestion
```

1. Log in to the Microsoft Azure Management Console.
2. Select the "Search resources, services, and docs" option at the top and search for "Storage accounts". </br> <img src="/resources/azure/storageaccounts/storage-account-has-tags/step2.png"/>
3. Select the "Storage account" by clicking on its "Name" to get into its configuration page.</br> <img src="/resources/azure/storageaccounts/storage-account-has-tags/step3.png"/> 
4. On the configuration page choose  "Tags" from the left navigation panel. </br> <img src="/resources/azure/storageaccounts/storage-account-has-tags/step4.png"/> 

> 5. On "Tags" panel enter the name/value of the tag you want to addand click "Save" on the bottom of the page.</br> <img src="/resources/azure/storageaccounts/storage-account-has-tags/step5.png"/> 

Reviewer: ```suggestion
5. On "Tags" panel enter the name/value of the tag you want to add and click "Save" on the bottom of the page.</br> <img src="/resources/azure/storageaccounts/storage-account-has-tags/step5.png"/> 
```
